lib/_debug_agent.js

```diff
@@ -111,6 +105,10 @@ Client.prototype.destroy = function destroy(msg) {
   this.emit('close');
 };
 
+const matchHeaderValue = /^([^:\s\r\n]+)\s*:\s*([^\s\r\n]+)\r\n/;
```

Comment:
Very tiny nit: I think it's more common to prefix with `re` or `regex` than `match`.